UnitTests/Utils/TestGlobals.cs

```diff
@@ -0,0 +1,8 @@
+namespace UnitTests
+{
+    public static class TestGlobals
+    {
+        public const string TEST_USER_ID = "TestUser";
+        public const string TEST_JWT_KEY = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c";
```

Comment:
What are the security implications of committing this JWT key to github?

Reply:
This is an example JWT token taken from https://jwt.io/

We just needed any JWT token for the tests to work, so this JWT token isn't real and doesn't contain any real information.